README.md

```diff
@@ -1,46 +1,160 @@
+| `sympa_bulk_sleep`                         | `1`                                                                                                                   | The number of seconds a bulk sleeps between starting a new loop if it  didn't find a message to send.                                                                                                                                                                                                                                    |
+| `sympa_bulk_wait_to_fork`                  | `10`                                                                                                                  | Number of seconds a master bulk waits between two packets number checks.                                                                                                                                                                                                                                                                 |
+| `sympa_default_max_list_members`           | `0`                                                                                                                   | Default limit for the number of subscribers per list (0 means no limit)                                                                                                                                                                                                                                                                  |
+| `sympa_spool`                              | `/var/spool/sympa`                                                                                                    | Directory containing various specialized spools                                                                                                                                                                                                                                                                                          |
+| `sympa_queue`                              | `/var/spool/sympa/msg`                                                                                                | Directory for message incoming spool                                                                                                                                                                                                                                                                                                     |
+| `sympa_queuemod`                           | `/var/spool/sympa/moderation`                                                                                         | Directory for moderation spool                                                                                                                                                                                                                                                                                                           |
```

Comment:
It would be nice if the role also created these directories. Otherwise I don't see a great benefit in leaving it configurable.